pom.xml

```diff
@@ -50,6 +50,13 @@
 			<artifactId>validation-api</artifactId>
 			<version>2.0.1.Final</version>
 		</dependency>
+
+		<!-- PostgreSQL Data Base -->
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<version>42.3.1</version>
+		</dependency>
```

Comment:
As far as I remember, you still doubt the choice of database.
Let's move this to the next commit, shall we?

Reply:
@VadimKhromenko, yes, you are probably right.
I removed the dependency.
Let's discuss again tomorrow the choice of database for our solution.

Reply:
@DanaKhromenko, sure, no problem.